dashboard/app/models/circuit_playground_discount_application.rb

```diff
@@ -136,7 +149,18 @@ def self.admin_application_status(user)
     # exception being if the user changed schools since confirming.
 
     {
-      is_pd_eligible: studio_person_pd_eligible?(user),
+      is_pd_eligible: studio_person_pd_eligible?(user,
+        [
+          Pd::Workshop::SUBJECT_SUMMER_WORKSHOP,
+        ]
+      ),
+      is_quarterly_workshop_pd_eligible: studio_person_pd_eligible?(user,
+        [
+          Pd::Workshop::SUBJECT_CSD_WORKSHOP_4,
+          Pd::Workshop::SUBJECT_CSD_WORKSHOP_6,
+          Pd::Workshop::SUBJECT_CSD_VIRTUAL_8
+        ]
+      ),
```

Comment:
I wonder if we can deduplicate these 12 lines with the matching ones in `self.application_status`?

Reply:
I put together a functional approach to to this (create a method that takes a user and creates a two element hash with keys `is_pd_eligible` and `is_quarterly_workshop_pd_eligible`, then merge that hash with the one created here). It seemed kind of funky / confusing though, is there a cleaner way to do that?